bin/aurcheck

```diff
@@ -76,6 +76,19 @@ namever_aur() {
     xargs -r aursearch -ir | jq -er '.results[] | [.Name, .PackageBase, .Version] | @tsv'
 }
 
+namever_vcs() {
+    for pkg in "$@"; do
+        base=$(aursearch -ir "$pkg" | jq -er '.results[] | .PackageBase')
+        cd "$HOME/.cache/aursync/$base" || continue   # TODO proper error handling, here and below
```

Comment:
I'll replace hardcoded value with `AURDEST` from aursync, but I want your advice on handling errors, what do you want to see - just ignore error and continue to the next package, print something to stderr, ... ?